cli/src/main/java/org/aya/cli/render/RenderOptions.java

```diff
@@ -73,6 +77,38 @@ public boolean isDefault() {
     return colorScheme.equals(DEFAULT_COLOR_SCHEME) && styleFamily.equals(DEFAULT_STYLE_FAMILY);
   }
 
+  public @NotNull String prettyColorScheme() {
+    return colorScheme == ColorSchemeName.Custom ? Objects.requireNonNull(path) : colorScheme.toString();
+  }
+
+  public @NotNull String prettyStyleFamily() {
+    return styleFamily.toString();
+  }
+
+  public void updateColorScheme(@NotNull Either<ColorSchemeName, Path> nameOrPath) throws IllegalArgumentException {
+    if (nameOrPath.isLeft()) {
+      var color = nameOrPath.getLeftValue();
+      if (color == ColorSchemeName.Custom) throw new IllegalArgumentException(
+        "To set a custom color scheme, just give the path to it :)");
+      this.colorScheme = color;
+    } else {
+      this.colorScheme = ColorSchemeName.Custom;
+      this.path = nameOrPath.getRightValue().toAbsolutePath().toString();
+    }
+    invalidate();
+  }
+
+  public void updateStyleFamily(@NotNull Either<StyleFamilyName, Path> nameOrPath) throws IllegalArgumentException {
+    if (nameOrPath.isRight()) throw new IllegalArgumentException("We don't support custom style family yet :)");
+    this.styleFamily = nameOrPath.getLeftValue();
+    invalidate();
+  }
+
+  public void invalidate() {
+    colorSchemeCache = null;
+    styleFamilyCache = null;
```

Comment:
Maybe we should invalidate them separately? However, this is okay for now because we can only change the color scheme.

Reply:
I don't think it is a good idea, as it makes the status of render options much more complicated.

Reply:
And we don't care about the extra performance overhead. It must not be the bottleneck.

Reply:
You persuaded me.